tests/framework/extensions/clusters/aks/importing/import_aks_cluster.go

```diff
@@ -0,0 +1,105 @@
+package importing
+
+import (
+	"context"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/cloudcredentials"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters/aks"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters/aks/resources"
+	"github.com/sirupsen/logrus"
+	kwait "k8s.io/apimachinery/pkg/util/wait"
+)
+
+var (
+	ctx                     = context.Background()
```

Comment:
Out of curiosity, is this the preferred context specifically for AKS? This won't cancel any context whether the aks client cancels that or not (assuming this context passed to the aks clients). I think context.TODO would be the best fit here if it's not specific. Also for future implementations, I think these should be instantiated in the function scopes, otherwise, the extension function call would overwrite it as they are global variables currently.

Reply:
agreed this needs to be a context.TODO, or else it would hang since there is no cancel.